design/design.md

```diff
@@ -0,0 +1,203 @@
+# CDM Spark Events Design
+
+## Nomenclature
+
+* CTS - the [CDM Task Service](https://github.com/kbase/cdm-task-service)
+
+## Purpose
+
+The CDM Spark events repo is responsible for automatically taking action based on upstream events.
+The initial target is processing output from the CTS, but additional event sources
+could be added in the future, e.g. responding to data uploaded to a dataplane bucket.
+
+The action taken depends on the nature of the event. For instance, if a CTS job completes and
+places job output in the data plane, the event system may run code to import data into CDM
+Deltalake tables.
```

Comment:
Can you add in concrete examples of jobs and the potential output that this service will be dealing with?

Reply:
Added